base/src/main/java/io/spine/base/Identifier.java

```diff
@@ -500,6 +567,8 @@ Message fromMessage(Message message) {
 
         abstract <I> I defaultValue(Class<I> idClass);
 
+        abstract boolean matchField(FieldDescriptor field);
```

Comment:
Let's use `FieldDeclaration` instead of a descriptor.

Reply:
As discussed vocally let's keep it as is to simplify filtering of fields.